src/Sylius/Behat/Resources/config/services/pages/admin/account.xml

```diff
@@ -13,8 +13,10 @@
 
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <services>
-        <defaults public="true" />
+        <defaults public="false" />
```

Comment:
Do we want to make all the behat services private? /cc @pamil

Reply:
I propose that for all the pages, not for the Behat context services. 
All services which implements Behat\Behat\Context\Context needs to be public.
Moreover, friends-of-behat/symfony-extension with autoconfigure make them public.

Reply:
@lchrusciel Most of them are already private, overriding the file defaults configuration:
- src/Sylius/Behat/Resources/config/services/pages/admin/admin_user.xml
- src/Sylius/Behat/Resources/config/services/pages/admin/channel.xml
- src/Sylius/Behat/Resources/config/services/pages/admin/country.xml